src/assets/styles/globals.css

```diff
@@ -25,7 +25,7 @@ va-button#mdFormButton::part(button) {
 }
 
 .feedback-button {
-  background-color: var(color-primary);
+  background-color: var(--color-primary);
```

Comment:
Webpack's CSS loader was apparently more forgiving than Turbopack's. This was the only problem I saw, though, so I think we're okay?